src/views/Dashboard.vue

```diff
@@ -162,7 +162,12 @@ export default {
   },
 
   computed: {
-    ...mapState(['entityIncNo'])
+    ...mapState(['entityIncNo']),
+
+    /** The NR Number string. */
+    nrNumber (): string {
+      return sessionStorage.getItem('NR_NUMBER')
+    }
```

Comment:
Hmmm, there is something very similar in `App.vue` but with a str.replace. How are these two methods applied differently?

Reply:
In the dashboard I don't care what format the string is, just whether it truthy or falsy.
In the other modules I may actually use the string, either for display (with a space) or for querying (with a %20).
So I kept it simple here.